include/vapor/ImageParams.h

```diff
@@ -22,9 +22,8 @@ class PARAMS_API ImageParams : public RenderParams {
     //
     virtual size_t GetRenderDim() const override { return (2); }
 
-    //
-    // Get and set image file path
-    //
+    //! Set image file path
```

Comment:
This is the path to the image file that will be read in and displayed, correct? Please clarify

Reply:
Clarified.